### DISCUSSION TITLE
Auto update and detection method

### DISCUSSION THREAD

**Discussion by phunghuan43** [svrooij/WinTuner#119] [Q&A]
Hello,

I encountered this great Wintuner by chance, after reading documentation, I have some questions:

-Would it be possible to enable parameter "-EnableAutoUpdate" in the app by "Update-WtIntuneApp" at the time it is initially packaged and deployed to Intune so that the apps could be auto updated without manually running 2-line script? This would be useful in scenario there are many apps imported from Wintuner to Intune, and we want to them to be kept up-to-date automatically?

- Is detection method working as similarly as it would when deploying app with LOB app?

**Comment by fedthans** [comment]
use Winget to do that job

**Comment by svrooij** [comment]
Starting with the latest version 1.3.0, this option is always turned on for each available assignment.
